src/utils/isAdmin.ts

```diff
@@ -0,0 +1,9 @@
+import useSessionProvider from '../hooks/useSessionProvider';
+
+export const isAdmin = (): boolean => {
+  const { session } = useSessionProvider();
```

Comment:
You use here hook so it's not utility but another hook. Also, you can make hone hook that returns user info (nickname avatar, etc) and includes info about the role

Reply:
I want to dig deeper on this outside of this mr